test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestOpenJ9DiagnosticsMXBean.java

```diff
@@ -488,14 +505,37 @@ private static boolean findAndDeleteFile(String dirPath, String fileName) throws
 		return found;
 	}
 
+
 	/**
+	 * Function to check if the specified system dump dataset is present
+	 *
+	 * @param actual_dsn System dump data set name
+	 * @return a boolean indicating if the system dump data set was found or not.
+	 */
+	private static boolean findAndDeleteDataset(String actual_dsn) throws ClassNotFoundException, NoSuchMethodException, IllegalAccessException, InvocationTargetException {
+		boolean found = false;
+		if (zOS) {
+			Class<?> zfileClass = Class.forName("com.ibm.jzos.ZFile");
```

Comment:
Won't this just (always) fail with ClassNotFoundException? As far as I'm aware, we don't yet have builds which include the jzos support.

Reply:
I don't see a zOS openjdk8-openj9 build on adopt as of now, will it be made available?

Reply:
Not any time soon. Perhaps in the future, still tbd.

Reply:
Thanks Peter.  Another option I can think of, is to copy the dataset to unix file system and verify using the File API instead of jzos APIs.  Would this be a better solution?

Reply:
Not sure there is any concern. IBM Java 8 builds will have com.ibm.jzos.ZFile. The test could be excluded for any other z/OS build until such time as com.ibm.jzos.ZFile is available. An alternative is to print a warning when the class cannot be found, and pass the core dump portion of the testing.